cli-typescript/src/utils/constants.ts

```diff
@@ -92,6 +93,7 @@ export const supportedChainNames: { [key in SUPPORTED_CHAINS]: string } = {
   [SUPPORTED_CHAINS.ABSTRACT]: 'abstract',
   [SUPPORTED_CHAINS.BERACHAIN]: 'berachain',
   [SUPPORTED_CHAINS.MONAD_TESTNET]: 'monadTestnet',
```

Comment:
I see we removed testnet in other places let's remove here too if no longer in use.